core/txpool/legacypool/legacypool.go

```diff
@@ -1711,6 +1735,7 @@ func (t *lookup) Remove(hash common.Hash) {
 	}
```

Comment:
I'm wondering whether the error two lines up ever happens, because in that case we would retain a weird authlist and a memleak

Reply:
I can't really see a way it's possible for this to happen, but might be a good enough reason to just track auths by hash instead of pointer.